lib/storage/metadata/mongoclient/LogConsumer.js

```diff
@@ -35,27 +36,29 @@ class LogConsumer {
      * @return {undefined}
     */
     connectMongo(done) {
-        MongoClient.connect(this._mongoUrl, {
+        const client = new MongoClient(this._mongoUrl, {
             replicaSet: this._replicaSet,
             useNewUrlParser: true,
-        },
-        (err, client) => {
-            if (err) {
-                this._logger.error('Unable to connect to MongoDB',
-                    { error: err });
-                return done(err);
-            }
+            useUnifiedTopology: true,
```

Comment:
This shouldn't be needed, I guess this one is deprecated and at least enabled by default, can we check?